spec/init_pg.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/sh
-
-USER = postgres
-DB = "clear_spec"
-
-psql -U $USER -C "DROP IF EXISTS DATABASE $DB; CREATE DATABASE $DB;"
-psql -U $USER -d $DB "CREATE TABLE users(id integer PRIMARY, first_name );"
```

Comment:
I couldn't find a purpose for this file, and the syntax errors makes me believe it's just cruft. Let me know if I'm off my mark for deleting it.